test/test.js

```diff
@@ -87,7 +87,7 @@ tape('hex encoding', function (t) {
 tape('call digest for more than MAX_UINT32 bits of data', function (t) {
   var _hash = crypto.createHash('sha1')
   var hash = new Sha1()
-  var bigData = Buffer.alloc(Math.pow(2, 32) / 8)
+  var bigData = Buffer.alloc((Math.pow(2, 32) + Math.pow(2,31))/ 8)
```

Comment:
Can this actually be changed to `0xffffffff` and `0x7fffffff` ?

Reply:
That would make 32nd bit as 0. Have changed it to 0x1ffffffff